docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-mqtt.md

```diff
@@ -97,23 +114,23 @@ There are two formats for the MQTT notifier integration:
 
 ### Payload fields
```

Comment:
Can we add a similar intro sentence here like below "Each alert instance has the following payload fields" or the equivalent that is needed here? thanks!